File: blog-dao/src/main/java/com/dao/KeywordsMapper.java
package com.dao;

import com.entity.Keywords;

public interface KeywordsMapper {
    int deleteByPrimaryKey(Integer kid);

    int insert(Keywords record);

    int insertSelective(Keywords record);

    Keywords selectByPrimaryKey(Integer kid);

    int updateByPrimaryKeySelective(Keywords record);

    int updateByPrimaryKey(Keywords record);
}

File: blog-dao/src/main/java/com/dao/KeywordsMapper.kt
package com.dao

import com.entity.Keywords

interface KeywordsMapper {
    fun deleteByPrimaryKey(kid: Int?): Int

    fun insert(record: Keywords): Int

    fun insertSelective(record: Keywords): Int

    fun selectByPrimaryKey(kid: Int?): Keywords

    fun updateByPrimaryKeySelective(record: Keywords): Int

    fun updateByPrimaryKey(record: Keywords): Int
}